package/base-files/files/sbin/opkg-keep

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+for file in /etc/sysupgrade.conf /usr/lib/opkg/status /etc/rc.local; do
+    cp $file $file.bak
+    echo $file.bak >> /etc/sysupgrade.conf
+done
+
+cat <<\EOF > /etc/rc.local
```

Comment:
`/etc/rc.local` is user property and should not be used, better stage a custom init script also please no self generating code